### ISSUE TITLE
MIT - PI Budget Worksheet in RFP

### ISSUE THREAD

**Issue by kdez** [ecsion/eSeaGrant-Client-Bug-Tracker#720]
In Budget Worksheets, as PI:  When I add a new worksheet, the system is forcing me to enter Justifications for lines with zero dollars. This is wrong. I only have to enter Justifications when I'm spending money.
[Worksheet.pdf](https://github.com/ecsion/eSeaGrant-Client-Bug-Tracker/files/1008768/Worksheet.pdf)

**Comment by kdez** [comment]
Also: the "Back" button on a completed budget worksheet should go to the worksheets list:

http://eseagrant.mit.edu/index.php?module=RFPProposal&view=Detail&record=34770&mode=showBudgetListingData&tab_label=Budgets&tab_label=Budgets

not the Budget Instructions page:

http://eseagrant.mit.edu/index.php?module=RFPProposal&view=Detail&record=34770&mode=showBudgetListingData&tab_label=Budgets&tab_label=Budgets

**Comment by kdez** [comment]
More weirdness with the budget worksheets in the way they indicate the status of PI/User entries.  If I enter a dollar amount, it may, or may not, prompt me to Save and it may, or may not, prompt me to enter a Justification. Example below: I entered a Justification for the first line of salary but it doesn’t show that I entered it. It *does * show that I entered a justification for “Expendable Supplies and Equipment.”

![image](https://cloud.githubusercontent.com/assets/17729848/26254333/34cac1e0-3c84-11e7-9a68-b19b28c675d5.png)
